vms/avm/network/gossip.go

```diff
@@ -152,6 +152,5 @@ func (g *gossipMempool) GetFilter() (bloom []byte, salt []byte, err error) {
 	g.lock.RLock()
 	defer g.lock.RUnlock()
 
-	bloomBytes, err := g.bloom.Bloom.MarshalBinary()
-	return bloomBytes, g.bloom.Salt[:], err
```

Comment:
This was a bug - the salt bytes returned here would be modified by calls to `ResetBloomFilterIfNeeded` and could result in racy behavior.